system/security/securesocket/securesocket.cpp

```diff
@@ -1765,3 +2001,93 @@ ISmartSocketFactory *createSecureSmartSocketFactory(const char *_socklist, bool
 {
     return new CSecureSmartSocketFactory(_socklist, _retry, _retryInterval, _dnsInterval);
 }
+
+class CSingletonSecureSocketConnection: public CSingletonSocketConnection
+{
+public:
+    Owned<ISecureSocketContext> secureContextClient;
+    Owned<ISecureSocketContext> secureContextServer;
+    int tlsLogLevel;
+
+    CSingletonSecureSocketConnection(SocketEndpoint &_ep)
+    {
+        ep = _ep;
+        state = Snone;
+        cancelling = false;
+        secureContextClient.setown(createSecureSocketContextSecret("local", ClientSocket));
+        secureContextServer.setown(createSecureSocketContextSecretSrv("local"));
+#ifdef _CONTAINERIZED
+        tlsLogLevel = queryComponentConfig().getPropInt("logging/@detail", SSLogMin);
+        if (tlsLogLevel > 100) // or 80 (DebugMsgThreshold) ?
```

Comment:
I don't think we should have any logging levels above 100 (as per way levels are defined in jlog.hpp). With the highest being ExtraneousMsgThreshold (90).
But, we've not been brave enough to turn default logging down below 100 anywhere.
So if this is particularly verbose, and would only be switched on to diagnose something, I think ok to say > 100 for now, with a comment saying should be ExtraneousMsgThreshold or similar high tracing level, but not until default logging level is changed from 100 to something more moderate.